src/utils/externalData/pricefeed.ts

```diff
@@ -0,0 +1,23 @@
+import type { Tokendata } from "types/bribedata.raw";
+import { getTokenPrice } from "./beetsBack";
+import { getCoingeckoCurrentPrice, getCoingeckoPrice } from "./coingecko";
+import { getRpcPrice } from "./liveRpcQueries";
+
+export async function getPrice(history: boolean, token: Tokendata, ts?: number): Promise<number> {
+  if (history) {
+    if (!ts) return 0;
```

Comment:
is `ts` timestamp? i would probably write that out

Reply:
included in next PR